knotx/conf/cogboard.conf

```diff
@@ -7,7 +7,6 @@ modules {
   userController = "com.cognifide.cogboard.config.controller.UserController"
   versionController = "com.cognifide.cogboard.config.controller.VersionController"
   httpClient = "com.cognifide.cogboard.http.HttpClient"
-  sshClient = "com.cognifide.cogboard.ssh.SSHClient"
```

Comment:
to powinno wylecieć?

Reply:
Tak, każda instancja widżetu korzysta teraz ze swojej instancji klasy `SSHClient`.